### DISCUSSION TITLE
NVS is not valid

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D-TFT#48] [Q&A] [answered]
[esp3d_settings.cpp:544] isValidSettingsNvs(): Expected ESP3D_TFT-V1.0.0 but got Invalid data
[esp3d_tft.cpp:93] begin(): NVS is not valid, need resetting
Guru Meditation Error: Core  0 panic'ed (Interrupt wdt timeout on CPU0). 

how can I solve it

**Comment by luc-github** [comment]
it should happen only the first time you flash the FW it should automaticaly reset the settings and restart the board
does the restart is a dead loop ?

**Reply by liqijian101** [reply]
Yes ,it is dead loop

**Reply by luc-github** [reply]
reflash by enabling verbose log to see if it show why it reset 
Did you do some change in code ?

On my side I have erased my tft eeprom using Arduino IDE and reflashed , and reset process worked well, I got a clean boot

Check if your partition.csv is correct , also do you have this only on one TFT or several ? I guess you have several TFT for your Devt

**Comment by liqijian101** [comment]
it is error at this code.
esp_err_t ret = esp_lcd_rgb_panel_set_pclk(disp_panel, 6 * 1000 * 1000);

**Reply by luc-github** [reply]
Your issue is on your board using  [st7796](https://github.com/luc-github/ESP3D-TFT/blob/main/hardware/ESP32S3_BZM_TFT35_GT911/components/st7796/st7796.c) ?
because it use i80 drivers and the code you mention is a patch for rgb panel not i80

**Reply by luc-github** [reply, marked as answer]
and this code is not present in your hadware repository so I am guessing you selected wrong hardware definition for your board

**Comment by luc-github** [comment]
@liqijian101 did you solved your issue ? I see your closed but no reason so I  want to be sure

**Comment by luc-github** [comment]
I guess yes
